examples/PySDM_examples/deJong_Azimi/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""
+box- and single-column coalescence-focused examples used to test new

```

Comment:
let's mention this new example also in the `examples/README.md` file?

Reply:
good idea